src/Compilers/CSharp/Test/Emit3/ExtensionTypeTests.cs

```diff
@@ -3198,7 +3228,45 @@ public class C<T> { }
         var comp = CreateCompilation(src, targetFramework: TargetFramework.Net70);
         comp.VerifyDiagnostics();
 
-        CompileAndVerify(comp, symbolValidator: validate, sourceSymbolValidator: validate, verify: Verification.FailsPEVerify);
+        var verifier = CompileAndVerify(comp, symbolValidator: validate, sourceSymbolValidator: validate, verify: Verification.FailsPEVerify);
+        // Note: we don't emit a DynamicAttribute on synthesized field
+        verifier.VerifyTypeIL("R", """
+.class public sequential ansi sealed beforefieldinit R
+	extends [System.Runtime]System.ValueType
+{
+	.custom instance void [System.Runtime]System.ObsoleteAttribute::.ctor(string, bool) = (
+		01 00 43 45 78 74 65 6e 73 69 6f 6e 20 74 79 70
+		65 73 20 61 72 65 20 6e 6f 74 20 73 75 70 70 6f
+		72 74 65 64 20 69 6e 20 74 68 69 73 20 76 65 72
+		73 69 6f 6e 20 6f 66 20 79 6f 75 72 20 63 6f 6d
+		70 69 6c 65 72 2e 01 00 00
+	)
+	.custom instance void [System.Runtime]System.Runtime.CompilerServices.CompilerFeatureRequiredAttribute::.ctor(string) = (
+		01 00 0e 45 78 74 65 6e 73 69 6f 6e 54 79 70 65
+		73 00 00
+	)
+	// Fields
+	.field private class C`1<object> '<UnderlyingInstance>$'
```

Comment:
> `'<UnderlyingInstance>$'`

Did you want to use `{{WellKnownMemberNames.ExtensionFieldName}}` here like in most other tests?